test/spec/modules/userId_spec.js

```diff
@@ -411,6 +412,7 @@ describe('User ID', function () {
             { name: 'amxId', value: {'amxId': 'amx-id-value-amx-id-value-amx-id-value'} },
             { name: 'pubCommonId', value: {'pubcid': 'pubCommon-id-value-pubCommon-id-value'} },
             { name: 'identityLink', value: {'idl_env': 'identityLink-id-value-identityLink-id-value'} },
+            { name: 'imppid', value: {'imppid': 'imppid-value-imppid-value-imppid-value'} },
```

Comment:
right now this test still expects      expect(window.googletag._ppid).to.equal('pubCommonidvaluepubCommonidvalue'); on line 423; i think you want a second test where line 410 has ppid: 'yoursource.com', and an  expect(window.googletag._ppid).to.equal('yourvalue');

Reply:
Added a test to make sure "imppid" is set correctly.